src/successModal/SuccessModal.unit.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { shallow, mount } from 'enzyme'
 
-import SuccessModal from 'successModal'
+import SuccessModal from './successModal'
```

Comment:
No idea what it wrong with that, but that the only thing I see that could output this error:

```
FAIL src/successModal/SuccessModal.unit.tsx
  ● Test suite failed to run
    Cannot find module './successModal' from 'SuccessModal.unit.tsx'
      1 | import React from 'react'
      2 | import { shallow, mount } from 'enzyme'
    > 3 | 
        | ^
      4 | import SuccessModal from './successModal'
      5 | 
      6 | const defaultProps = {
      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:259:17)
      at Object.<anonymous> (src/successModal/SuccessModal.unit.tsx:3:1)
```